templates/article/overview_list.html.twig

```diff
@@ -0,0 +1,24 @@
+{% extends 'base.html.twig' %}
+
+{% block heading %} Обзоры {% endblock %}
+
+{% block content %}
+    {% if articles %}
+        {% for article in articles %}
+            <div class="card my-2">
+                <h4 class="card-title p-2 brand-txt">
+                    {{article.name}} <br>
+                </h4>
+                <div class="card-body px-1 py-0 d-flex flex-column">
+                    <p class="card-text m-2">{{article.description | cut}}</p>
```

Comment:
@daria-sieroshtan 
Don't you want to provide cut length?

Reply:
Default value is applicable